src/node/rpc/member_frontend.h

```diff
@@ -875,6 +875,7 @@ namespace ccf
         js::Context context(rt);
         rt.add_ccf_classdefs();
         js::TxContext txctx{&tx, js::TxAccess::GOV_RO};
+        js::populate_global_console(context);
```

Comment:
Do we really need this?

Reply:
Definitely not, but it's been useful to debug things. Maybe it should be turned off for all release builds?